### DISCUSSION TITLE
Argocd inside argocd

### DISCUSSION THREAD

**Discussion by tuatara73** [argoproj/argo-cd#11476] [Q&A]
We installed Argocd management and we wanted to install argocd-qa inside it that will be managed by the main one (management argocd) 
And we placed it on a different namespace.
We then added another argocd-dev on a different namespace and when we start using it, then it showed us errors and deleted stuff from the argocd-qa when we grant it `clusterAdminAccess: True` 
My main question, does Argocd support this approach in the first place and we somehow didn't configured it correctly?
If it's a bug, then let me know and I'll open a bug about it.
Thanks

**Comment by kostis-codefresh** [comment]
You need to provide some details on how you did each installation.

In general Argo CD only manages clusters (either internal or external). It does not manage other Argo CD installations. So there is no concept of "Argo CD inside Argo CD"

You should be able to install multiple Argo CD installations on a single cluster (each one constrained on a different namespace). But in this case each Argo CD installation is not related to any other (and there is no Argo CD installation that manages other installations)
